docs-src/typescript/how-to-send-a-query-to-a-workflow-in-typescript.md

```diff
@@ -0,0 +1,14 @@
+---
+id: how-to-send-a-query-to-a-workflow-in-typescript
+title: How to send a Query to a Workflow in TypeScript
+sidebar_label: Send a Query
+description: Use `WorkflowHandle.query` to query a running or completed Workflow.
+tags:
+  - typescript
+  - developer-guide
+---
+
+Use [`WorkflowHandle.query`](https://typescript.temporal.io/api/interfaces/client.workflowhandle/#query) to query a running or completed Workflow.
```

Comment:
This API is actually for "handling a Query" as far as I can tell - 
IMO - really poor choice of naming here... as "defining a Query" is really about defining the data structure of the Query information itself...

Need to update this to show that the data structure and name are independent of the method handling it.

Reply:
this API method and snippet is about sending

![image](https://user-images.githubusercontent.com/251288/192863333-131032d9-e6f8-4592-a3be-bf1c2cd8f202.png)